docs/csi-operator-1.10.0.yaml

```diff
@@ -0,0 +1,982 @@
+# This manifest deploys the OpenEBS CSI control plane components,
+# with associated CRs & RBAC rules. This manifest has been verified
+# only with Ubuntu 16.04 and CentOS hosts, due to dependencies on
```

Comment:
In CentOS-7.7 version, there is some problem with iscsi module right? Should we need to specify the supported versions?

Reply:
its supported, about the modules are not loading properly those steps we have in our docs